misc/bazel/rust_deps/BUILD.bazel

```diff
@@ -0,0 +1,31 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License in the LICENSE file at the
+# root of this repository, or online at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@bazel_skylib//rules:build_test.bzl", "build_test")
+
+build_test(
+    name = "build_test",
+    targets = [
+        "@cxxbridge//:cxx",
+        "@crates_io//:openssl",
+        "@crates_io//:protobuf-native",
+        "@crates_io//:rocksdb",
+        "@crates_io//:rdkafka",
+        "@crates_io//:tikv-jemallocator",
+        "@crates_io//:tokio",
```

Comment:
should this be here?

Reply:
Ahh! No it doesn't need `cpp`, it's just a relatively large crate that has interesting dependencies (e.g. proc_macros) that I want to make sure built correctly